backend/compat.py

```diff
@@ -1 +1,13 @@
+# Werk van Sil
 
+# for-loop
+# [] 
+
+
+def validate_appointment(data: dict):
+    KeyList = ['title', 'description', ...]
+    for key in KeyList:
+        if key not in dict:
```

Comment:
De `item in iterable` syntax checkt of een item onderdeel is van een collectie/iterables met data.

De `key in dictionary` syntax checkt of de `key` zich in de `dict` bevindt. Hoe zou je checken of een bepaalde variabele zich bevindt in de `values` van de dictionary? Hoe doe je dit met de `in` syntax?